nubis/puppet/nagios.pp

```diff
@@ -0,0 +1,63 @@
+cron::daily { "${project_name}-nagios":
+  hour    => '7',
+  minute  => fqdn_rand(60),
+  user    => 'etl',
+  command => "nubis-cron ${project_name}-nagios /opt/etl/nagios/run",
+}
+
+package { 'pyodbc':
```

Comment:
package definitions **must** be unique globally, so this needs to be moved nubis/puppet/1_packages.pp if not already there